src/backend/dx11/src/conv.rs

```diff
@@ -319,3 +414,46 @@ pub fn map_stage(stage: Stage) -> spirv::ExecutionModel {
         Stage::Domain => spirv::ExecutionModel::TessellationEvaluation,
     }
 }
+
+pub fn map_wrapping(wrap: WrapMode) -> D3D11_TEXTURE_ADDRESS_MODE {
+    match wrap {
+        WrapMode::Tile   => D3D11_TEXTURE_ADDRESS_WRAP,
+        WrapMode::Mirror => D3D11_TEXTURE_ADDRESS_MIRROR,
+        WrapMode::Clamp  => D3D11_TEXTURE_ADDRESS_CLAMP,
+        WrapMode::Border => D3D11_TEXTURE_ADDRESS_BORDER,
+    }
+}
+
+pub fn map_anisotropic(anisotropic: Anisotropic) -> D3D11_FILTER {
+    match anisotropic {
+        Anisotropic::On(_) => D3D11_FILTER_ANISOTROPIC,
+        Anisotropic::Off => 0,
```

Comment:
0 feels awkward here. Perhaps we could make this cleaner?

Reply:
This function is only used to construct the `D3D11_FILTER` enum through bit twiddling so I think 0 is fine here (see https://msdn.microsoft.com/en-us/library/windows/desktop/dn770367(v=vs.85).aspx#code-snippet-2).

Though considering it doesn't actually map to an enum like the other functions do in the module, might be good to rename it.

Reply:
this function shouldn't know the only way it's used, technically :) if it can't produce a `D3D11_FILTER` under some circumstances, it should return `Option`

Reply:
It is only used for the bit stuff, so I'd consider just renaming it to `map_anisotropic_mask` or something similar